README.md

```diff
@@ -20,3 +22,23 @@ Example using the [example_add_i32](https://github.com/stellar/rs-stellar-contra
 $ stellar-contract-cli invoke --file example_add_i32.wasm --fn add --arg i32:1 --arg i32:2
 i32:3
 ```
+
+### Calling a host function
+
+For both options below, the file should contain a JSON representation of the `SCVec` xdr object, which is passed directly to `invoke_function` in the host. The order of the items in the `SCVec` is specified below.
+
+#### Creating a contract
+
+```
+// args = SCVec(contractCode, salt, ed25519, signature)
+stellar-contract-cli invoke --file <JSONARGS> create-contract
+```
+
+The input file is the 
+
+#### Invoking an existing contract
+
+```
+// args = SCVec(contractID, functionName, args...)
+stellar-contract-cli invoke --file <JSONARGS> call
```

Comment:
Similar to comment to above, I think we should avoid overloading `--file` with anything other than a WASM file. This is ideally `cli invoke --file <WASM> --fn <...>` when a file is being passed as a short hand for deploy and execute, and then in the manual deploy case `cli invoke --contract-id <...> --fn <...>`. i.e. `--file` and `--contract-id` are mutually exclusive.